### DISCUSSION TITLE
Is this a bug?

### DISCUSSION THREAD

**Discussion by gsoec** [saleor/saleor#10920] [Q&A]
[                    instance.quantity,](https://github.com/saleor/saleor/blob/8a4924ad57c6ad97f820b511a060b22725ab38e3/saleor/graphql/order/mutations/order_line_update.py#L81)
Is this line should be `cleaned_input["quantity"]` ?

**Comment by peelar** [comment]
Hello @gsoec 👋 

Could you please elaborate on why do you think this is a bug?
